scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,9 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    brent does not ensure that the located minimum is within the initially
```

Comment:
Hm. Confusion about this issue continues to occur, so I'd say that the notes could use a thoughtful discussion about what the function is doing on the inside (in more detail than is currently given). 

The short summary

> Given a function of one-variable and a possible bracketing interval, return the minimum of the function isolated to a fractional precision of tol.

could also be improved. So I'd say there's still lots of room for improvement on this docstring if you're interested.